jsforce/jsforce
# ci: use typedoc for docs

Adds `publish-docs` workflow to build documentation from code using typedoc.

**TODO**
* set `v2-docs` branch as for GH pages here (https://github.com/jsforce/jsforce/settings/pages)

How to test:
Go to https://github.com/jsforce/jsforce/actions and run the `publish-docs` workflow, site should be up here:
https://jsforce.github.io/jsforce/ (this is already deployed from previous testing, check the `v2-docs` branch to see it actually committed the changes).

**This PR also removes CircleCI config file**
our GHA workflow already covers browser/node test scenarios

@W-9127316@
